EventVisualisation/Base/include/EventVisualisationBase/GeometryManager.h

```diff
@@ -39,7 +39,14 @@ class GeometryManager
   /// Returns ROOT shapes describing simplified geometry of given detector
   TEveGeoShape* getGeometryForDetector(std::string detectorName);
 
+  /// sets geometry to R2/R3
```

Comment:
Isn't this limiting to allow only exactly 2 versions of geometry (Run2 vs Run3)? What about a function
`setGeometryVersion(GeometryVersionEnum)` which might be more flexible?